xoinvader/tests/test_enemy_wave.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from xoinvader.enemy_wave import EnemyWave
+
+
+def test_wave():
+    e = EnemyWave()
+
+    a = 0
+    b = 0
+    def inc_a():
```

Comment:
adding nonlocal is unnescessary complication here, I think. Your code works with some other game code that in bright future I hope gets good or at least unified object system. Just write little mock here, please.